src/base/static/js/views/survey-view.js

```diff
@@ -135,5 +156,27 @@
       evt.preventDefault();
       this.$('textarea, input').not('[type="hidden"]').first().focus();
       Util.log('USER', 'place', 'leave-reply-btn-click', this.collection.options.placeModel.getLoggingDetails(), this.collection.size());
+    },
+
+    onUpdateResponse: function(evt) {
+      var cid = $(evt.target).parent().data("cid"),
+      model = this.collection.get(cid),
+      $form = $(evt.target).siblings("form"),
+      attrs = Util.getAttrs($form);
+      model.set(attrs).save({}, {
+        success: function() {
+          $(evt.target).addClass("faded").prop("disabled", true);
+        }
+      });
+    },
+
+    onDeleteResponse: function(evt) {
+      var response = confirm("You are deleting this comment permanently. Are you sure you want to continue?");
```

Comment:
No... do we have a way to localize content directly in js files? I'm actually not sure